Description: Relax version restrictions on required packages to those in GHC 7.4
Author: Iain Lane <laney@debian.org>
Forwarded: not-needed

Index: darcs/darcs.cabal
===================================================================
--- darcs.orig/darcs.cabal	2012-03-01 11:39:16.000000000 +0000
+++ darcs/darcs.cabal	2012-03-01 11:50:23.144609628 +0000
@@ -123,7 +123,7 @@
   else
     buildable: True

-    build-tools: ghc >= 6.10 && < 7.2
+    build-tools: ghc >= 6.10

     hs-source-dirs:   src
     include-dirs:     src
@@ -173,17 +173,17 @@
                      mtl          >= 1.0 && < 2.1,
                      parsec       >= 2.0 && < 3.2,
                      html         == 1.0.*,
-                     filepath     >= 1.1.0.0 && < 1.3.0.0,
+                     filepath     >= 1.1.0.0 && < 1.4.0.0,
                      haskeline    >= 0.6.2.2 && < 0.7,
                      hashed-storage >= 0.5.2 && < 0.6,
                      base >= 3,
                      bytestring >= 0.9.0 && < 0.10,
                      text == 0.11.*,
-                     old-time   == 1.0.*,
+                     old-time   == 1.1.*,
                      directory  >= 1.0.0.0 && < 1.2.0.0,
-                     process    == 1.0.*,
+                     process    == 1.1.*,
                      containers >= 0.1 && < 0.5,
-                     array      >= 0.1 && < 0.4,
+                     array      >= 0.1 && < 0.5,
                      random     == 1.0.*,
                      tar          == 0.3.*,
                      zlib >= 0.5.1.0 && < 0.6.0.0,
@@ -191,7 +191,7 @@
                      test-framework             >= 0.2.2,
                      test-framework-quickcheck2 >= 0.2.2
     if !os(windows)
-      build-depends: unix >= 1.0 && < 2.5
+      build-depends: unix >= 1.0 && < 2.6
     if flag(http)
         build-depends:    network >= 2.2 && < 2.4,
                           HTTP    >= 3000.0 && < 4000.2
@@ -207,7 +207,7 @@
   else
     buildable: True

-    build-tools: ghc >= 6.10 && < 7.2
+    build-tools: ghc >= 6.10

     hs-source-dirs:   src
     include-dirs:     src
@@ -378,22 +378,22 @@
                      mtl          >= 1.0 && < 2.1,
                      parsec       >= 2.0 && < 3.2,
                      html         == 1.0.*,
-                     filepath     >= 1.1.0.0 && < 1.3.0.0,
+                     filepath     >= 1.1.0.0 && < 1.4.0.0,
                      haskeline    >= 0.6.2.2 && < 0.7,
                      hashed-storage >= 0.5.2 && < 0.6,
                      tar          == 0.3.*

     if !os(windows)
-      build-depends: unix >= 1.0 && < 2.5
+      build-depends: unix >= 1.0 && < 2.6

     build-depends: base >= 3,
                    bytestring >= 0.9.0 && < 0.10,
                    text == 0.11.*,
-                   old-time   == 1.0.*,
+                   old-time   == 1.1.*,
                    directory  >= 1.0.0.0 && < 1.2.0.0,
-                   process    == 1.0.*,
+                   process    == 1.1.*,
                    containers >= 0.1 && < 0.5,
-                   array      >= 0.1 && < 0.4,
+                   array      >= 0.1 && < 0.5,
                    random     == 1.0.*


@@ -482,7 +482,7 @@
                     src/umask.c
                     src/Crypt/sha2.c

-  build-tools: ghc >= 6.10 && < 7.2
+  build-tools: ghc >= 6.10

   -- We need optimizations, regardless of what Hackage says
   -- Note: "if true" works around a cabal bug with order of flag composition
@@ -526,22 +526,22 @@
                    mtl          >= 1.0 && < 2.1,
                    parsec       >= 2.0 && < 3.2,
                    html         == 1.0.*,
-                   filepath     >= 1.1.0.0 && < 1.3.0.0,
+                   filepath     >= 1.1.0.0 && < 1.4.0.0,
                    haskeline    >= 0.6.2.2 && < 0.7,
                    hashed-storage >= 0.5.2 && < 0.6,
                    tar          == 0.3.*

   if !os(windows)
-    build-depends: unix >= 1.0 && < 2.5
+    build-depends: unix >= 1.0 && < 2.6

   build-depends: base >= 3,
                  bytestring >= 0.9.0 && < 0.10,
                  text == 0.11.*,
-                   old-time   == 1.0.*,
+                   old-time   == 1.1.*,
                    directory  >= 1.0.0.0 && < 1.2.0.0,
-                   process    == 1.0.*,
+                   process    == 1.1.*,
                    containers >= 0.1 && < 0.5,
-                   array      >= 0.1 && < 0.4,
+                   array      >= 0.1 && < 0.5,
                    random     == 1.0.*

   if flag(curl)
@@ -614,7 +614,7 @@
   else
     buildable: True

-    build-tools: ghc >= 6.10 && < 7.2
+    build-tools: ghc >= 6.10

     build-depends:   base          < 5,
                      extensible-exceptions >= 0.1 && < 0.2,
@@ -622,7 +622,7 @@
                      mtl          >= 1.0 && < 2.1,
                      parsec       >= 2.0 && < 3.2,
                      html         == 1.0.*,
-                     filepath     >= 1.1.0.0 && < 1.3.0.0,
+                     filepath     >= 1.1.0.0 && < 1.4.0.0,
                      QuickCheck   >= 2.1.0.0,
                      HUnit        >= 1.0,
                      test-framework             >= 0.2.2,
@@ -683,11 +683,11 @@
                    bytestring >= 0.9.0 && < 0.10,
                    haskeline    >= 0.6.2.2 && < 0.7,
                    text == 0.11.*,
-                   old-time   == 1.0.*,
+                   old-time   == 1.1.*,
                    directory  >= 1.0.0.0 && < 1.2.0.0,
-                   process    == 1.0.*,
+                   process    == 1.1.*,
                    containers >= 0.1 && < 0.5,
-                   array      >= 0.1 && < 0.4,
+                   array      >= 0.1 && < 0.5,
                    hashed-storage >= 0.5.2 && < 0.6,
                    random     == 1.0.*
